### DISCUSSION TITLE
Underscore character "_" to create colon character ":" in HTML

### DISCUSSION THREAD

**Discussion by DerpGeraltOfRivia** [Knio/dominate#162] [Q&A]
For the application I am creating, I need xmlns:o in the HTML tag. I couldn't find anything about the colon in the documentation, and discovered by pure chance that typing

      _html = html(xmlns_o)

creates the tag

      <html xmlns:o>

My question is: will this ever change in future updates to the library? It is vitally important that my HTML is structured in this way.

Also, I would appreciate if this was part of the documentation, as it took me a couple hours of experimentation to discover this.

**Comment by Knio** [comment]
This feature should not change
